contracts/mechs/token/BalanceTrackerFixedPriceToken.sol

```diff
@@ -68,16 +67,8 @@ contract BalanceTrackerFixedPriceToken is BalanceTrackerFixedPriceBase {
     /// @param amount Token amount.
     /// @return Received amount.
     function _getRequiredFunds(address requester, uint256 amount) internal virtual override returns (uint256) {
```

Comment:
Do we have a similar implementation for non-olas tokens? or is that not a usecase we need?

I'm thinking USDC

Reply:
yes suggest we do that